UnityProject/Assets/Scripts/Objects/Closets/ClosetControl.cs

```diff
@@ -709,6 +709,20 @@ private void OpenItemHandling()
 			serverHeldItems.Clear();
 		}
 
+		private void CheckPlayerCrawlState(ObjectBehaviour player)
+		{
+			var regPlayer = player.GetComponent<RegisterPlayer>();
+			Debug.Log(regPlayer.IsLayingDown);
```

Comment:
Debug.Log!

Reply:
Huh? I deleted that from my repo and it's not on my end in any list of changes.
huh, weird.

Reply:
Fixed.

Reply:
if you do need to leave a debug statement one day, remember to use ``Logger`` class instead

Reply:
No need. I forgot to push the commit that removes the Debug.log and now it's removed.
We can merge now.